src/main/kotlin/no/nav/amt/arena/acl/services/GjennomforingService.kt

```diff
@@ -1,47 +0,0 @@
-package no.nav.amt.arena.acl.services
-
-import no.nav.amt.arena.acl.domain.Gjennomforing
-import no.nav.amt.arena.acl.repositories.GjennomforingDbo
-import no.nav.amt.arena.acl.repositories.GjennomforingRepository
-import org.springframework.stereotype.Service
-import java.util.*
-
-val SUPPORTED_TILTAK = setOf(
-	"INDOPPFAG",
-	"ARBFORB",
-	"AVKLARAG",
-	"VASV",
-	"ARBRRHDAG",
-	"DIGIOPPARB",
-	"JOBBK",
-	"GRUPPEAMO",
-	"GRUFAGYRKE"
-)
-
-@Service
-class GjennomforingService(
```

Comment:
Ser ikke helt gevinsten med å fjerne en service for at consumeren skal bruke repository direkte. Vi har ofte forsøkt at man holder kontrollen over repository til servicer. Fordelen er kanskje ikke så synlig i akkurat dette tilfellet men samtidig så er det allerede oppdelt jeg tenker er ryddig og tilrettelagt for fremtidige endringer

Reply:
Service-lag som kun delegerer til neste lag, er et anti-pattern som har blitt tatt opp i bl.a. teknisk retning.